### ISSUE TITLE
Importing functions from sliced OpenAPI document fails

### ISSUE THREAD

**Issue by zengin** [microsoftgraph/semantic-kernel#20]
- Generate APIManifest for the following path: `/me/messages`
- Observe that plugin import succeeds
- Generate APIManifest for the following paths, one APIManifest plugin for each:
  - "/drive/root/children/{driveItem-id}"
  - "/me"
  - "/me/calendar"

- Observe a serialization error:
```
Unhandled exception. System.AggregateException: Plugin creation failed for DriveItemPlugin ('N' is an invalid start of a value. Path: $ | LineNumber: 2092 | BytePositionInLine: 62.)
 ---> System.Text.Json.JsonException: 'N' is an invalid start of a value. Path: $ | LineNumber: 2092 | BytePositionInLine: 62.
 ---> System.Text.Json.JsonReaderException: 'N' is an invalid start of a value. LineNumber: 2092 | BytePositionInLine: 62.
   at System.Text.Json.ThrowHelper.ThrowJsonReaderException(Utf8JsonReader& json, ExceptionResource resource, Byte nextByte, ReadOnlySpan`1 bytes)
   at System.Text.Json.Utf8JsonReader.ConsumeValue(Byte marker)
   at System.Text.Json.Utf8JsonReader.ConsumeNextToken(Byte marker)
   at System.Text.Json.Utf8JsonReader.ConsumeNextTokenOrRollback(Byte marker)
   at System.Text.Json.Utf8JsonReader.ReadSingleSegment()
   at System.Text.Json.Utf8JsonReader.Read()
   at System.Text.Json.Utf8JsonReader.TrySkip()
   at System.Text.Json.JsonDocument.TryParseValue(Utf8JsonReader& reader, JsonDocument& document, Boolean shouldThrow, Boolean useArrayPools)
   at System.Text.Json.JsonElement.ParseValue(Utf8JsonReader& reader)
   at System.Text.Json.Serialization.Converters.JsonArrayConverter.ReadList(Utf8JsonReader& reader, Nullable`1 options)
   at System.Text.Json.Serialization.Converters.JsonArrayConverter.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at Json.More.JsonSerializerOptionsExtensions.Read[T](JsonSerializerOptions options, Utf8JsonReader& reader)
   at Json.Schema.EnumKeywordJsonConverter.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at Json.Schema.JsonSerializerOptionsExtensions.ArbitraryDeserializer`1.Read(Utf8JsonReader& reader, JsonSerializerOptions options)
   at Json.Schema.SchemaJsonConverter.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at System.Text.Json.Serialization.JsonConverter`1.TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value, Boolean& isPopulatedValue)
   at System.Text.Json.Serialization.JsonCollectionConverter`2.OnTryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, TCollection& value)
   at System.Text.Json.Serialization.JsonConverter`1.TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value, Boolean& isPopulatedValue)
   at System.Text.Json.Serialization.JsonResumableConverter`1.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at Json.More.JsonSerializerOptionsExtensions.Read[T](JsonSerializerOptions options, Utf8JsonReader& reader)
   at Json.Schema.OneOfKeywordJsonConverter.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at Json.Schema.JsonSerializerOptionsExtensions.ArbitraryDeserializer`1.Read(Utf8JsonReader& reader, JsonSerializerOptions options)
   at Json.Schema.SchemaJsonConverter.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at System.Text.Json.Serialization.JsonConverter`1.TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value, Boolean& isPopulatedValue)
   at System.Text.Json.Serialization.JsonDictionaryConverter`3.OnTryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, TDictionary& value)
   at System.Text.Json.Serialization.JsonConverter`1.TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value, Boolean& isPopulatedValue)
   at System.Text.Json.Serialization.JsonResumableConverter`1.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at Json.More.JsonSerializerOptionsExtensions.Read[T](JsonSerializerOptions options, Utf8JsonReader& reader)
   at Json.Schema.PropertiesKeywordJsonConverter.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at Json.Schema.JsonSerializerOptionsExtensions.ArbitraryDeserializer`1.Read(Utf8JsonReader& reader, JsonSerializerOptions options)
   at Json.Schema.SchemaJsonConverter.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at System.Text.Json.Serialization.JsonConverter`1.TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value, Boolean& isPopulatedValue)
   at System.Text.Json.Serialization.JsonCollectionConverter`2.OnTryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, TCollection& value)
   at System.Text.Json.Serialization.JsonConverter`1.TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value, Boolean& isPopulatedValue)
   at System.Text.Json.Serialization.JsonResumableConverter`1.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at Json.More.JsonSerializerOptionsExtensions.Read[T](JsonSerializerOptions options, Utf8JsonReader& reader)
   at Json.Schema.AnyOfKeywordJsonConverter.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at Json.Schema.JsonSerializerOptionsExtensions.ArbitraryDeserializer`1.Read(Utf8JsonReader& reader, JsonSerializerOptions options)
   at Json.Schema.SchemaJsonConverter.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at System.Text.Json.Serialization.JsonConverter`1.TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value, Boolean& isPopulatedValue)
   at System.Text.Json.Serialization.JsonDictionaryConverter`3.OnTryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, TDictionary& value)
   at System.Text.Json.Serialization.JsonConverter`1.TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value, Boolean& isPopulatedValue)
   at System.Text.Json.Serialization.JsonResumableConverter`1.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at Json.More.JsonSerializerOptionsExtensions.Read[T](JsonSerializerOptions options, Utf8JsonReader& reader)
   at Json.Schema.PropertiesKeywordJsonConverter.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at Json.Schema.JsonSerializerOptionsExtensions.ArbitraryDeserializer`1.Read(Utf8JsonReader& reader, JsonSerializerOptions options)
   at Json.Schema.SchemaJsonConverter.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at System.Text.Json.Serialization.JsonConverter`1.TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value, Boolean& isPopulatedValue)
   at System.Text.Json.Serialization.JsonCollectionConverter`2.OnTryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, TCollection& value)
   at System.Text.Json.Serialization.JsonConverter`1.TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value, Boolean& isPopulatedValue)
   at System.Text.Json.Serialization.JsonResumableConverter`1.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at Json.More.JsonSerializerOptionsExtensions.Read[T](JsonSerializerOptions options, Utf8JsonReader& reader)
   at Json.Schema.AllOfKeywordJsonConverter.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at Json.Schema.JsonSerializerOptionsExtensions.ArbitraryDeserializer`1.Read(Utf8JsonReader& reader, JsonSerializerOptions options)
   at Json.Schema.SchemaJsonConverter.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at System.Text.Json.Serialization.JsonConverter`1.TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value, Boolean& isPopulatedValue)
   at System.Text.Json.Serialization.JsonCollectionConverter`2.OnTryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, TCollection& value)
   at System.Text.Json.Serialization.JsonConverter`1.TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value, Boolean& isPopulatedValue)
   at System.Text.Json.Serialization.JsonResumableConverter`1.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at Json.More.JsonSerializerOptionsExtensions.Read[T](JsonSerializerOptions options, Utf8JsonReader& reader)
   at Json.Schema.AllOfKeywordJsonConverter.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at Json.Schema.JsonSerializerOptionsExtensions.ArbitraryDeserializer`1.Read(Utf8JsonReader& reader, JsonSerializerOptions options)
   at Json.Schema.SchemaJsonConverter.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at Json.More.JsonSerializerOptionsExtensions.Read[T](JsonSerializerOptions options, Utf8JsonReader& reader)
   at Json.Schema.ItemsKeywordJsonConverter.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at Json.Schema.JsonSerializerOptionsExtensions.ArbitraryDeserializer`1.Read(Utf8JsonReader& reader, JsonSerializerOptions options)
   at Json.Schema.SchemaJsonConverter.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at System.Text.Json.Serialization.JsonConverter`1.TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value, Boolean& isPopulatedValue)
   at System.Text.Json.Serialization.JsonDictionaryConverter`3.OnTryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, TDictionary& value)
   at System.Text.Json.Serialization.JsonConverter`1.TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value, Boolean& isPopulatedValue)
   at System.Text.Json.Serialization.JsonResumableConverter`1.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at Json.More.JsonSerializerOptionsExtensions.Read[T](JsonSerializerOptions options, Utf8JsonReader& reader)
   at Json.Schema.PropertiesKeywordJsonConverter.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at Json.Schema.JsonSerializerOptionsExtensions.ArbitraryDeserializer`1.Read(Utf8JsonReader& reader, JsonSerializerOptions options)
   at Json.Schema.SchemaJsonConverter.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at System.Text.Json.Serialization.JsonConverter`1.TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value, Boolean& isPopulatedValue)
   at System.Text.Json.Serialization.JsonCollectionConverter`2.OnTryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, TCollection& value)
   at System.Text.Json.Serialization.JsonConverter`1.TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value, Boolean& isPopulatedValue)
   at System.Text.Json.Serialization.JsonResumableConverter`1.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at Json.More.JsonSerializerOptionsExtensions.Read[T](JsonSerializerOptions options, Utf8JsonReader& reader)
   at Json.Schema.AnyOfKeywordJsonConverter.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at Json.Schema.JsonSerializerOptionsExtensions.ArbitraryDeserializer`1.Read(Utf8JsonReader& reader, JsonSerializerOptions options)
   at Json.Schema.SchemaJsonConverter.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at System.Text.Json.Serialization.JsonConverter`1.TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value, Boolean& isPopulatedValue)
   at System.Text.Json.Serialization.JsonDictionaryConverter`3.OnTryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, TDictionary& value)
   at System.Text.Json.Serialization.JsonConverter`1.TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value, Boolean& isPopulatedValue)
   at System.Text.Json.Serialization.JsonResumableConverter`1.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at Json.More.JsonSerializerOptionsExtensions.Read[T](JsonSerializerOptions options, Utf8JsonReader& reader)
   at Json.Schema.PropertiesKeywordJsonConverter.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at Json.Schema.JsonSerializerOptionsExtensions.ArbitraryDeserializer`1.Read(Utf8JsonReader& reader, JsonSerializerOptions options)
   at Json.Schema.SchemaJsonConverter.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at System.Text.Json.Serialization.JsonConverter`1.TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value, Boolean& isPopulatedValue)
   at System.Text.Json.Serialization.JsonCollectionConverter`2.OnTryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, TCollection& value)
   at System.Text.Json.Serialization.JsonConverter`1.TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value, Boolean& isPopulatedValue)
   at System.Text.Json.Serialization.JsonResumableConverter`1.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at Json.More.JsonSerializerOptionsExtensions.Read[T](JsonSerializerOptions options, Utf8JsonReader& reader)
   at Json.Schema.AllOfKeywordJsonConverter.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at Json.Schema.JsonSerializerOptionsExtensions.ArbitraryDeserializer`1.Read(Utf8JsonReader& reader, JsonSerializerOptions options)
   at Json.Schema.SchemaJsonConverter.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at System.Text.Json.Serialization.JsonConverter`1.TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value, Boolean& isPopulatedValue)
   at System.Text.Json.Serialization.JsonCollectionConverter`2.OnTryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, TCollection& value)
   at System.Text.Json.Serialization.JsonConverter`1.TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value, Boolean& isPopulatedValue)
   at System.Text.Json.Serialization.JsonResumableConverter`1.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at Json.More.JsonSerializerOptionsExtensions.Read[T](JsonSerializerOptions options, Utf8JsonReader& reader)
   at Json.Schema.AnyOfKeywordJsonConverter.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at Json.Schema.JsonSerializerOptionsExtensions.ArbitraryDeserializer`1.Read(Utf8JsonReader& reader, JsonSerializerOptions options)
   at Json.Schema.SchemaJsonConverter.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at System.Text.Json.Serialization.JsonConverter`1.TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value, Boolean& isPopulatedValue)
   at System.Text.Json.Serialization.JsonDictionaryConverter`3.OnTryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, TDictionary& value)
   at System.Text.Json.Serialization.JsonConverter`1.TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value, Boolean& isPopulatedValue)
   at System.Text.Json.Serialization.JsonResumableConverter`1.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at Json.More.JsonSerializerOptionsExtensions.Read[T](JsonSerializerOptions options, Utf8JsonReader& reader)
   at Json.Schema.PropertiesKeywordJsonConverter.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at Json.Schema.JsonSerializerOptionsExtensions.ArbitraryDeserializer`1.Read(Utf8JsonReader& reader, JsonSerializerOptions options)
   at Json.Schema.SchemaJsonConverter.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at System.Text.Json.Serialization.JsonConverter`1.TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value, Boolean& isPopulatedValue)
   at System.Text.Json.Serialization.JsonCollectionConverter`2.OnTryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, TCollection& value)
   at System.Text.Json.Serialization.JsonConverter`1.TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value, Boolean& isPopulatedValue)
   at System.Text.Json.Serialization.JsonResumableConverter`1.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at Json.More.JsonSerializerOptionsExtensions.Read[T](JsonSerializerOptions options, Utf8JsonReader& reader)
   at Json.Schema.AllOfKeywordJsonConverter.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at Json.Schema.JsonSerializerOptionsExtensions.ArbitraryDeserializer`1.Read(Utf8JsonReader& reader, JsonSerializerOptions options)
   at Json.Schema.SchemaJsonConverter.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at System.Text.Json.Serialization.JsonConverter`1.TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value, Boolean& isPopulatedValue)
   at System.Text.Json.Serialization.JsonCollectionConverter`2.OnTryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, TCollection& value)
   at System.Text.Json.Serialization.JsonConverter`1.TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value, Boolean& isPopulatedValue)
   at System.Text.Json.Serialization.JsonResumableConverter`1.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at Json.More.JsonSerializerOptionsExtensions.Read[T](JsonSerializerOptions options, Utf8JsonReader& reader)
   at Json.Schema.AllOfKeywordJsonConverter.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at Json.Schema.JsonSerializerOptionsExtensions.ArbitraryDeserializer`1.Read(Utf8JsonReader& reader, JsonSerializerOptions options)
   at Json.Schema.SchemaJsonConverter.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at System.Text.Json.Serialization.JsonConverter`1.TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value, Boolean& isPopulatedValue)
   at System.Text.Json.Serialization.JsonConverter`1.ReadCore(Utf8JsonReader& reader, JsonSerializerOptions options, ReadStack& state)
   --- End of inner exception stack trace ---
   at System.Text.Json.ThrowHelper.ReThrowWithPath(ReadStack& state, JsonReaderException ex)
   at System.Text.Json.Serialization.JsonConverter`1.ReadCore(Utf8JsonReader& reader, JsonSerializerOptions options, ReadStack& state)
   at System.Text.Json.Serialization.Metadata.JsonTypeInfo`1.Deserialize(Utf8JsonReader& reader, ReadStack& state)
   at System.Text.Json.JsonSerializer.ReadFromSpan[TValue](ReadOnlySpan`1 utf8Json, JsonTypeInfo`1 jsonTypeInfo, Nullable`1 actualByteCount)
   at System.Text.Json.JsonSerializer.ReadFromSpan[TValue](ReadOnlySpan`1 json, JsonTypeInfo`1 jsonTypeInfo)
   at System.Text.Json.JsonSerializer.Deserialize[TValue](String json, JsonSerializerOptions options)
   at Json.Schema.JsonSchema.FromText(String jsonText, JsonSerializerOptions options)
   at Microsoft.SemanticKernel.KernelJsonSchema.Parse(String jsonSchema) in C:\github\microsoftgraph\semantic-kernel\dotnet\src\SemanticKernel.Abstractions\Functions\KernelJsonSchema.cs:line 33
   at Microsoft.SemanticKernel.Plugins.OpenApi.OpenApiSchemaExtensions.ToJsonSchema(OpenApiSchema schema) in C:\github\microsoftgraph\semantic-kernel\dotnet\src\Functions\Functions.OpenApi\Extensions\OpenApiSchemaExtensions.cs:line 23
   at Microsoft.SemanticKernel.Plugins.OpenApi.OpenApiDocumentParser.CreateRestApiOperationExpectedResponses(OpenApiResponses responses)+MoveNext() in C:\github\microsoftgraph\semantic-kernel\dotnet\src\Functions\Functions.OpenApi\OpenApi\OpenApiDocumentParser.cs:line 276
   at System.Linq.Enumerable.ToDictionary[TSource,TKey,TElement](IEnumerable`1 source, Func`2 keySelector, Func`2 elementSelector, IEqualityComparer`1 comparer)
   at System.Linq.Enumerable.ToDictionary[TSource,TKey,TElement](IEnumerable`1 source, Func`2 keySelector, Func`2 elementSelector)
   at Microsoft.SemanticKernel.Plugins.OpenApi.OpenApiDocumentParser.CreateRestApiOperations(String serverUrl, String path, OpenApiPathItem pathItem, IList`1 operationsToExclude) in C:\github\microsoftgraph\semantic-kernel\dotnet\src\Functions\Functions.OpenApi\OpenApi\OpenApiDocumentParser.cs:line 187
   at Microsoft.SemanticKernel.Plugins.OpenApi.OpenApiKernelExtensions.CreatePluginFromApiManifestAsync(Kernel kernel, String pluginName, String apiManifestFilePath, OpenApiFunctionExecutionParameters executionParameters, CancellationToken cancellationToken) in C:\github\microsoftgraph\semantic-kernel\dotnet\src\Functions\Functions.OpenApi\Extensions\OpenApiKernelExtensions.cs:line 307
   at Microsoft.SemanticKernel.Plugins.OpenApi.OpenApiKernelExtensions.ImportPluginFromApiManifestAsync(Kernel kernel, String pluginName, String filePath, OpenApiFunctionExecutionParameters executionParameters, CancellationToken cancellationToken) in C:\github\microsoftgraph\semantic-kernel\dotnet\src\Functions\Functions.OpenApi\Extensions\OpenApiKernelExtensions.cs:line 88
   at ApiManifestPluginsExample.Program.AddApiManifestPluginsAsync(Kernel sk, IConfigurationRoot configuration) in C:\github\microsoftgraph\semantic-kernel\dotnet\samples\ApiManifestPluginsExample\Program.cs:line 141
   --- End of inner exception stack trace ---
   at ApiManifestPluginsExample.Program.AddApiManifestPluginsAsync(Kernel sk, IConfigurationRoot configuration) in C:\github\microsoftgraph\semantic-kernel\dotnet\samples\ApiManifestPluginsExample\Program.cs:line 153
   at ApiManifestPluginsExample.Program.Main(String[] args) in C:\github\microsoftgraph\semantic-kernel\dotnet\samples\ApiManifestPluginsExample\Program.cs:line 61
   at ApiManifestPluginsExample.Program.<Main>(String[] args)
```

### Expected Behavior
Plugin imports succeed.

### Related
- [x] #27

**Comment by zengin** [comment]
- [ ] Create a copy of sample OpenAPI yaml without this bug.

**Comment by peombwa** [comment]
Fixed by #42.
